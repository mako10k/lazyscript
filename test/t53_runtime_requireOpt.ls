!{
  let res = !requireOpt "no_such_file.ls";
  let res2 = !requireOpt "lib_req.ls";
  (~Prelude .to_str) (~Prelude .Option res) ~res ++ "|" ++ (~Prelude .to_str) (~Prelude .Option res2) ~res2
};
