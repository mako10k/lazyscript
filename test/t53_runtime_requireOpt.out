OK-from-lib
"None|Some ()"
